ko/convolutional-neural-networks.md

```diff
@@ -1,716 +1,715 @@
 **Convolutional Neural Networks translation**
 
-<br>
+<br> 
 
 **1. Convolutional Neural Networks cheatsheet**
 
-&#10230;
+&#10230; 합성곱 신경망 치트시트
 
 <br>
 
 
 **2. CS 230 - Deep Learning**
 
-&#10230;
+&#10230; CS 230 - 딥러닝
 
 <br>
 
 
 **3. [Overview, Architecture structure]**
 
-&#10230;
+&#10230; [개요, 구조]
 
 <br>
 
 
 **4. [Types of layer, Convolution, Pooling, Fully connected]**
 
-&#10230;
+&#10230; [레이어, 합성곱, 풀링, 풀리커넥티드]
 
 <br>
 
 
 **5. [Filter hyperparameters, Dimensions, Stride, Padding]**
 
-&#10230;
+&#10230; [필터 하이퍼파라미터, 차원, 스트라이드, 패딩]
 
 <br>
 
 
 **6. [Tuning hyperparameters, Parameter compatibility, Model complexity, Receptive field]**
 
-&#10230;
+&#10230; [하이퍼파라미터 튜닝, 파라미터 호환성, 모델 복잡성, 수용 필드]
 
 <br>
 
 
 **7. [Activation functions, Rectified Linear Unit, Softmax]**
 
-&#10230;
+&#10230; [활성화 함수, 수정된 선형 유닛, 소프트맥스]
```

Comment:
If we really wanted to translate ReLU, I think 정류선형함수 would be a more accurate translation... however, I think the term ReLU is commonly used as-is—there really isn't a direct translation probably because there isn't really a need for it, as is the case with many CS/math terms